tests/test_plugin.py

```diff
@@ -33,3 +45,31 @@ def test_destroy(fxa_client, fxa_account):
     result = testdir.runpytest()
     result.assert_outcomes(error=1, passed=1)
     assert 'ClientError: Unknown account' in result.stdout.str()
+
+
+def test_commandline_email_option(testdir, monkeypatch, random_email):
+    monkeypatch.setenv('TEST_EMAIL', '{}'.format(random_email))
+    testdir.makepyfile("""
+        import os
+
+        import pytest
+
+        def test_account(fxa_account):
+            assert os.getenv('TEST_EMAIL') == fxa_account.email
```

Comment:
I would use string interpolation to inject the value of `random_email` here instead of taking it from the environment variable.